### DISCUSSION TITLE
Canvas LTI 1.3 failing with 404

### DISCUSSION THREAD

**Discussion by AlexGellert** [community/community#40096] [GitHub Education] [answered]
Documentation was wrong, I put up a documentation PR

**Comment by jfullstackdev** [comment, marked as answer]
what do you mean by that ? is this about GitHub ?

**Reply by AlexGellert** [reply]
the OIDC and token endpoints are incorrect for Canvas LTI 1.3 installation of github classroom, they just need to be swapped.

should be updated soon.

Please close this issue :)

![image](https://user-images.githubusercontent.com/84880439/204104021-f2a18204-a311-4069-8445-6a1adff2acb2.png)
